package.json

```diff
@@ -39,10 +39,10 @@
     "supertest": "1.2.0"
   },
   "scripts": {
-    "pretest": "npm run lint",
+    "posttest": "npm run lint",
```

Comment:
I prefer having linter before tests because the checks are quick and usually trivial. In place, we define a script `mocha` to launch only the tests.